python/grass/grassdb/checks.py

```diff
@@ -139,6 +139,21 @@ def get_mapset_owner(mapset_path):
         return None
 
 
+def get_mapset_history_path(mapset_path):
```

Comment:
This returns gui history, but this function is not meant only for gui, there is also .bash_history, so maybe 2 functions or a parameter is needed.